### DISCUSSION TITLE
第三者レビュー 環境切り替え-開発ガイド

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#571] [General]
#### ビルドバリアントごとのアプリ起動

- 「Firebaseの機能を利用する」ではiOSが左、Androidが右だったのに、こちらでは逆になっていたのが気になりました
- 縦の並び順が「Firebaseの機能を利用する」と違うのが少し気になりました

**Comment by creature-water-valley** [comment]
### 対応内容

- iOS、Android順に修正
- 「Firebaseの機能を利用する」の順番にあわせる

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/558

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
